docs/swagger.yaml

```diff
@@ -1660,6 +1660,56 @@ paths:
         in: path
         required: true
         type: string
+  /courses/v1/block_metadata/{usage_key_string}:
+    get:
+      operationId: courses_v1_block_metadata_read
+      summary: '**Use Case**'
+      description: |-
+        Returns the block metadata. Data like index_dictionary related to a
+            block should be fetched using this API, because they are too large for
+            the cache used by the course blocks/transformers API.
+
+        **Example requests**:
+
+            GET /api/courses/v1/block_metadata/<usage_id>/?
+                &include=index_dictionary
+
+        **Parameters**:
+
+            * "include": a comma-separated list of keys to include.
+              Valid keys are "index_dictionary".
+
+        **Response Values**
+
+            A dictionary containing:
+            * id (string): Block usage_key_str.
+            * type (string) Block type.
+            * index_dictionary: (dict) The index_dictionary JSON data
+              (usually this is the text content of the block, for search
+              indexing or other purposes) for this block. Returned only if
+              the "index_dictionary" is included in the "include"
+              parameter.
+      parameters:
+        - name: page
+          in: query
+          description: A page number within the paginated result set.
+          required: false
+          type: integer
+        - name: page_size
+          in: query
+          description: Number of results to return per page.
+          required: false
+          type: integer
```

Comment:
I was going to merge this just now, but I noticed this minor issue with the docs... there are no `page` and `page_size` parameters right? The only query parameter is `include`

Reply:
@bradenmacdonald  Ohh yes, swagger is adding page and page_size to list APIs, missed to remove it. Thanks!